projects/common/src/index.ts

```diff
@@ -23,11 +23,11 @@ export const articleTypes = [
 export type PillarFromPalette = typeof articlePillars[number]
 export type ArticleType = typeof articleTypes[number]
 
-interface ColorAppearance {
+export interface ColorAppearance {
```

Comment:
Can we see if we can delete this. @walaura do you know where any objects of this type are constructed / do we still use it?

Reply:
This needs design to consider. Merging as is.